async-download-rest/src/main/java/org/uniprot/api/async/download/messaging/producer/map/MapJobSubmissionRules.java

```diff
@@ -0,0 +1,14 @@
+package org.uniprot.api.async.download.messaging.producer.map;
+
+import org.uniprot.api.async.download.messaging.producer.JobSubmissionRules;
```

Comment:
Do we need two classes UniProtKBMapJobSubmissionRules
        and MapJobSubmissionRules